### PR TITLE
Update QDs to QL 1

### PR DESCRIPTION
**Pull request by brawner**
Signed-off-by: Stephen Brawner <brawner@gmail.com>

### CODE REVIEW

File: geometry_msgs/QUALITY_DECLARATION.md [hunk 1]
```diff
@@ -82,15 +82,15 @@ There are no currently copyrighted source files in this package.
```
Comment (chapulina) [inline geometry_msgs/QUALITY_DECLARATION.md:82]: I noticed that only `4.v` is listed inside testing. Maybe we should list all items?

(the same goes for several other packages)

**Review by chapulina — COMMENTED** [review]
Bookkeeping - these PRs need to go in first:

* https://github.com/ros2/rcl_interfaces/pull/116
* https://github.com/ros2/rosidl_defaults/pull/15

File: geometry_msgs/QUALITY_DECLARATION.md (hunk 1 above)
Comment (brawner) [reply geometry_msgs/QUALITY_DECLARATION.md:82]: They would all say the same thing. This is an interfaces package and doesn't require them. It seemed simpler to just write that at the top of section 4

Comment (chapulina) [reply geometry_msgs/QUALITY_DECLARATION.md:82]: Fair enough, sounds good

**Review by chapulina — APPROVED** [review]
LGTM, but let's try to merge the PRs in order to make sure we're not missing anything.

**Pull request merged**